results.tex

```diff
@@ -85,33 +89,28 @@ \subsection{Nuclear Material Inventory}
 \end{table}
 
 
-\Cref{tab:pu} lists the isotope, mass fraction,
-and quantity of plutonium that can be obtained from the 2050 \gls{UNF} inventory.
-
 
 \subsection{French \gls{SFR} Deployment}
 
 Reprocessing \gls{UNF} collected from all EU nations can start approximately
-180 \glspl{SFR}. With the \gls{SFR} breeding ratio of over one, France can transition into
+180 \glspl{SFR}. Table \ref{tab:pu} lsits the isotope, mass fraction,
+and quantity of plutonium that can be obtained from the 2050 \gls{UNF} inventory.
+ With the \gls{SFR} breeding ratio of over one, France can transition into
 a fully \gls{SFR} fleet without extra construction of \glspl{LWR}. 
 
 From Varaine et al. \cite{varaine_pre-conceptual_2012}, a French
 ASTRID-type \gls{SFR} of capacity 600 \gls{MWe} needs $1.225$ tons of
 plutonium a year, with an initial plutonium loading of $4.9$ tons. 
 Thus, the number of \glspl{SFR} that can be loaded with the reprocessed
 plutonium from \gls{UNF} can be estimated to $\frac{Pu \ from \ legacy \ \gls{UNF}}{4.9} \approx 181$ \glspl{SFR},
-assuming infinite reprocessing and fabrication capacity as well as
-abundant depleted uranium supply. 
-
-Also, assuming that \gls{MOX} can be recycled indefinitely,
-used \gls{MOX} from an ASTRID reactor contains enough plutonium to produce a \gls{MOX} fuel with
-the same mass, if mixed with depleted uranium. For example,
-used \gls{MOX} from an ASTRID reactor is assumed to be 23.95\% plutonium
+assuming abundant depleted uranium supply. 
+ 
+Used \gls{MOX} from an ASTRID reactor is assumed to be 23.95\% plutonium
 in this simulation (see \cref{tab:comp}), whereas a fresh \gls{MOX} is 22\% plutonium.
-Separating plutonium from used \gls{MOX} from
-an ASTRID reactor can create \gls{MOX} of the mass of used \gls{MOX}.
 The plutonium breeding ratio in this simulation is thus assumed to be
 $\approx 1.08$.
+Separating plutonium from used \gls{MOX} from
```

Comment:
This reads like a riddle. "Separating plutonium from used MOX from an ASTRID reactor can create MOX of the mass of used MOX."

What is this sentence trying to communicate?

Reply:
I was trying to say that separating the spent fuel provides enough material to make more of that same fuel, but now I find this entire sentence redundant, after mentioning it has a breeding ratio over one. I'll erase it!